tests/cpp/test_multidevice_overlap.cpp

```diff
@@ -932,6 +932,22 @@ TEST_F(AllgatherOverlapTest, AllgatherBasedPipeliningHostIrImplementation) {
 }
 
 class RingAllgatherOverlapTest : public MultiDeviceTest {
+ private:
+  float kernel_time_ms_ = 0;
+  float avg_time_per_iter_ = 0;
+  cudaEvent_t start_event_ = {};
+  cudaEvent_t finish_event_ = {};
```

Comment:
```suggestion
  cudaEvent_t start_event_;
  cudaEvent_t finish_event_;
```
Does this trigger false lint warnings?